--- pvfs-1.5.6/pvfs-kernel-1.5.6-linux-2.4/Makefile.in.org	Fri Nov 15 17:26:54 2002
+++ pvfs-1.5.6/pvfs-kernel-1.5.6-linux-2.4/Makefile.in	Sat Dec 14 13:01:57 2002
@@ -45,7 +45,7 @@
 INCLUDES=-I. -I$(srcdir)/pvfs-includes -I$(srcdir)/libkernsys/include
 KINCLUDES=-I. -I$(srcdir)/libkpvfs/kinclude -I$(srcdir)/libkpvfs/klib -I$(srcdir)/libkpvfs/kshared -I$(srcdir)/libkernsys/include
 CFLAGS=-MMD -Wall -Winline -Wstrict-prototypes -DLINUX \
-	$(PVFS_HEADER_PATH) $(INCLUDES) @PVFS_ARCH_CFLAGS@
+	$(PVFS_HEADER_PATH) $(INCLUDES) $(SMPFLAGS) @PVFS_ARCH_CFLAGS@
 
 MODFLAGS=-MMD @PVFS_KERNEL_HEADER_PATH@ $(KINCLUDES)\
 	@PVFS_WORKAROUND_CFLAGS@ -O2 -Wall \
